### PR TITLE
Refs #29514: Add puma status

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
Repoclosures are all related to mass EL8 rebuilds happening

**Comment by sthirugn** [comment]
I created a foreman issue last week to track this change - https://projects.theforeman.org/issues/29514

File: packages/foreman/rubygem-net_http_unix/rubygem-net_http_unix.spec [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+cp -a .%{_bindir}/* \
+        %{buildroot}%{_bindir}/
+find %{buildroot}%{gem_instdir}/bin -type f | xargs chmod a+x
+
+%files
+%dir %{gem_instdir}
+%{_bindir}/net_http_unix
+%exclude %{gem_instdir}/.gitignore
+%exclude %{gem_instdir}/.travis.yml
+%{gem_instdir}/Guardfile
```
Comment (evgeni) [inline packages/foreman/rubygem-net_http_unix/rubygem-net_http_unix.spec:78]: can be excluded

File: gem2rpm/scl.spec.erb [hunk 2]
```diff
@@ -4,6 +4,7 @@ require 'json'
 root = `git rev-parse --show-toplevel`.strip
 SCL_PREFIXES = JSON.load(File.read(File.join(root, 'scl_prefixes.json')))
 LICENSES = JSON.load(File.read(File.join(root, 'licenses.json')))
+config.rules[:ignore].push(*['Guardfile'])
```
Comment (ehelms) [inline gem2rpm/scl.spec.erb:7]: @evgeni Added a commit that adds Guardfile to our ignore list from here on out.

Comment (ekohl) [reply gem2rpm/scl.spec.erb:7]: We should do that more often. There's more files like that that we often end up ignoring.

Comment (ehelms) [reply gem2rpm/scl.spec.erb:7]: Agreed. That's why I laid the groundwork.

Comment (evgeni) [reply gem2rpm/scl.spec.erb:7]: @ehelms could you update to the syntax Ewoud posted from the other template?

Comment (ehelms) [reply gem2rpm/scl.spec.erb]: The irony is, I started with the syntax and the voice in my head said why repeat yourself. >.< Updated.

**Pull request merged**